modules/thermal_hydraulics/src/components/HeatTransfer1PhaseBase.C

```diff
@@ -18,6 +18,7 @@ HeatTransfer1PhaseBase::validParams()
 {
   InputParameters params = HeatTransferBase::validParams();
   params.addParam<FunctionName>("Hw", "Convective heat transfer coefficient [W/(m^2-K)]");
+  params.declareControllable("Hw");
```

Comment:
The same comment as for the friction factor above

Reply:
Similar reply as for friction factor; see `Closures1PhaseSimple::addMooseObjectsHeatTransfer`

Reply:
Similar reply as for friction factor above :-)